### ISSUE TITLE
Undefined method `match' in `multiline_pipe.rb`

### ISSUE THREAD

**Issue by michaelherold** [brigade/haml-lint#200]
Originally reported here: michaelherold/codeclimate-haml_lint#2 by @andyw8

```
/usr/local/bundle/gems/haml_lint-0.23.1/lib/haml_lint/linter/multiline_pipe.rb:42:in `check': undefined method `match' for nil:NilClass (NoMethodError)
Did you mean?  catch
	from /usr/local/bundle/gems/haml_lint-0.23.1/lib/haml_lint/linter/multiline_pipe.rb:20:in `visit_silent_script'
	from /usr/local/bundle/gems/haml_lint-0.23.1/lib/haml_lint/haml_visitor.rb:40:in `safe_send'
	from /usr/local/bundle/gems/haml_lint-0.23.1/lib/haml_lint/haml_visitor.rb:19:in `visit'
	from /usr/local/bundle/gems/haml_lint-0.23.1/lib/haml_lint/haml_visitor.rb:30:in `block in visit_children'
	from /usr/local/bundle/gems/haml_lint-0.23.1/lib/haml_lint/haml_visitor.rb:30:in `each'
	from /usr/local/bundle/gems/haml_lint-0.23.1/lib/haml_lint/haml_visitor.rb:30:in `visit_children'
	from /usr/local/bundle/gems/haml_lint-0.23.1/lib/haml_lint/haml_visitor.rb:24:in `visit'
	from /usr/local/bundle/gems/haml_lint-0.23.1/lib/haml_lint/linter.rb:28:in `run'
	from /usr/local/bundle/gems/haml_lint-0.23.1/lib/haml_lint/runner.rb:84:in `block in collect_lints'
	from /usr/local/bundle/gems/haml_lint-0.23.1/lib/haml_lint/runner.rb:83:in `map'
	from /usr/local/bundle/gems/haml_lint-0.23.1/lib/haml_lint/runner.rb:83:in `collect_lints'
	from /usr/local/bundle/gems/haml_lint-0.23.1/lib/haml_lint/runner.rb:119:in `process_file'
	from /usr/local/bundle/gems/haml_lint-0.23.1/lib/haml_lint/runner.rb:108:in `block in process_files'
	from /usr/local/bundle/gems/haml_lint-0.23.1/lib/haml_lint/runner.rb:107:in `each'
	from /usr/local/bundle/gems/haml_lint-0.23.1/lib/haml_lint/runner.rb:107:in `process_files'
	from /usr/local/bundle/gems/haml_lint-0.23.1/lib/haml_lint/runner.rb:132:in `report'
	from /usr/local/bundle/gems/haml_lint-0.23.1/lib/haml_lint/runner.rb:23:in `run'
	from /usr/src/app/lib/cc/engine/source_file.rb:64:in `process'
	from /usr/src/app/lib/cc/engine/haml_lint.rb:54:in `block in run'
	from /usr/src/app/lib/cc/engine/haml_lint.rb:48:in `run'
	from /usr/src/app/bin/codeclimate-haml_lint:10:in `<main>'
```

@andyw8 do you happen to know what the file is that it's blow up on? I'll be happy to work on a fix, I just need some reproduction steps.

It should be a file that uses `|` to split between lines if that gives you any clues.

**Comment by andyw8** [comment]
@michaelherold I found it was caused by a single file containing Windows-style line endings. Once I fixed that, it ran without any problem.

**Comment by michaelherold** [comment]
Awesome, thanks. I believe that was fixed in 0.23.2 (which I just pushed in the CodeClimate engine this morning).

I'm going to close this bug as related to #195. Please let us know if it pops up again.
